spec/features/user_index_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+RSpec.describe 'user_index', type: :feature do
+  before(:each) do
+    @user = User.first
+    visit users_path
+  end
+
+  it 'shows the username of the users' do
+    expect(page).to have_content('Tom')
+  end
+
+  it 'shows the photos of the users' do
+    expect(page.html).to include('avatal')
+  end
+
+  it 'shows the number of post of each user' do
+    expect(page).to have_content("Number of Posts: #{@user.posts_counter}")
+  end
```

Comment:
- [ ] For the user index page, you still need to include a test for this assertion

  ![red-](https://user-images.githubusercontent.com/36542986/186380642-9c2ebc3c-8f30-4502-8ae6-d1fcebe04818.jpg)

Reply:
@TedLivist 
can invite you for a call

Reply:
@mandelatuks alright